modules/installation-aws-edge-compute-pools-examples.adoc

```diff
@@ -0,0 +1,79 @@
+// Module included in the following assemblies:
+// * installing/installing-aws-localzone.adoc (Installing a cluster on AWS with worker nodes on AWS Local Zones)
+// * installing/installing-aws-wavelength-zone.adoc (Installing a cluster on AWS with worker nodes on AWS Wavelength Zones) 
+
+:_mod-docs-content-type: CONCEPT
+[id="installation-aws-edge-compute-pools-examples_{context}"]
+= Examples of installation configuration files with edge compute pools
+
+The following examples show `install-config.yaml` files that contain an edge machine pool configuration.
+
+.Configuration that uses an edge pool with a custom instance type
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: devcluster.openshift.com
+metadata:
+  name: ipi-edgezone
+compute:
+- name: edge
+  platform:
+    aws:
+      type: r5.2xlarge
+platform:
+  aws:
+    region: us-west-2
+pullSecret: '{"auths": ...}'
+sshKey: ssh-ed25519 AAAA...
+----
+
+Instance types differ between locations. To verify availability in the {zone-type} in which the cluster runs, see the AWS documentation.
+
+.Configuration that uses an edge pool with a custom Amazon Elastic Block Store (EBS) type
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: devcluster.openshift.com
+metadata:
+  name: ipi-edgezone
+compute:
+- name: edge
+  platform:
+    aws:
+      zones:
+      - us-west-2-lax-1a
+      - us-west-2-lax-1b
+      - us-west-2-phx-2a
+      rootVolume:
+        type: gp3
+        size: 120
+platform:
+  aws:
+    region: us-west-2
+pullSecret: '{"auths": ...}'
+sshKey: ssh-ed25519 AAAA...
+----
+
+Elastic Block Storage (EBS) types differ between locations. Check the AWS documentation to verify availability in the {zone-type} in which the cluster runs.
```

Comment:
nit: Do we need an ifdef including only Local Zones? it is not valid for WL.

Reply:
Updated